python/pyspark/sql/connect/client/core.py

```diff
@@ -1335,6 +1370,16 @@ def handle_response(
                     with attempt:
                         for b in self._stub.ExecutePlan(req, metadata=self._builder.metadata()):
                             yield from handle_response(b)
+        except KeyboardInterrupt:
+            logger.debug(f"Interrupt request received for operation={req.operation_id}")
+            try:
+                self.interrupt_operation(req.operation_id)
+            except Exception as e:
+                # Swallow all errors if aborted.
+                logger.debug(f"Caught an error during interrupt handling, silenced: {e}")
+                pass
```

Comment:
nit remove

Reply:
since this is only debug logging, might be worthwhile to keep to make it easier to debug special cases where the keyboard interrupt is fired and the query cancellation fails?